project/i-padiolleau/README.rst

```diff
@@ -0,0 +1,49 @@
+##Basic Information
+https://github.com/i-padiolleau/optimization-for-composite-type-data-structure
+
+##Problem to Solve
+When it comes to optimize a code that manipulate at signicant amount of data, it is important to consider how the data is access to adapt the data structure used in the code. 
+There is different solutions to store composite types array and the data structure impact the efficiency of the cache loading it.
+- AoS : Arrays of composed data structure
+- SoA : Composite structure composed of different array types
+- Single array with maching index and different type 
+
+##Prospective Users
+The user will be able to manipulate the test that demonstrate the performance of each approch considering optimization
+
+##System Architecture
+The input gived by the user will be parameter for the tests, to change size of arrays, types and test in order to produces graphics results to show to the user.44
+
+##API Description
+The test.py file wile provided some pre-programmed tests that the user could choose to runs and all the functions that could be used to write a script (creation of the variable, allocation of memory, function that used data in order to test data locality)
+
+##Engineering Infrastructure
+**Build** will be perform by **Makefile** with "make" command
+**Git will** serve as **version control** : Each week will be marked as V+1.0 and modified features marked as VX.+1
+**Testing** will be perform using **python3.pytest**
+**Documentation** (comments and **README** file to explain the few steps to use the code)
+
+##Schedule
+Week 1 (10/31):
+Surrend proposal and worked the problem on paper, to some research online about data locality and tests demonstrating this principle
+
+Week 2 (11/7):
+Make the structure of the C++ code, Makefile

```

Comment:
Without specific problems, it is very difficult to come up with code or data structures.

Reply:
Its corrected